user_guide/common_features.rst

```diff
@@ -0,0 +1,79 @@
+.. _user-guide-common-features:
+
+Common Features
+===============
+
+Customers, Channels and Accounts
+--------------------------------
+
+One of the main tasks for any CRM is ability to aggregate the details of a customer. 
+For example, you have two web-based stores, a retail outlet and a factory. There is a company whose managers 
+purchase from your web-based stores and with whom you have some business arrangements at the 
+factory. 
```

Comment:
Comment: The syntax for this example is a little confusing. Are you saying that company A has a web-based retail outlet and a web-based factory? And company B, which is a customer of company A, employs managers who purchase from both of company A's webstores (in addition to the business arrangements company B has with company A's factory)? This should all be clarified. More importantly, however, this example needs to be elaborated on somewhat so that it adequately helps explain a CRM's "ability to aggregate the details of a customer." This is currently not that clear.